Plutarch/Api/V1/Value.hs

```diff
@@ -41,6 +41,9 @@ newtype PValue (s :: S) = PValue (Term s (PMap PCurrencySymbol (PMap PTokenName
     (PlutusType, PIsData)
     via (DerivePNewtype PValue (PMap PCurrencySymbol (PMap PTokenName PInteger)))
 
+instance PEq PValue where
```

Comment:
Now that I think about it, we sehould use the same `PEq` for `PBuiltinList (PAsData a)`, and then we can just use newtype derive here.

Reply:
Correction:
@L-as I don't think that this instance will help because 
```haskell
type PBuiltinMap = (PBuiltinList (PBuiltinPair (PAsData a) (PAsData b))```
and hence it picks the usual instance for `PEq` that isn't `PData`.